### DISCUSSION TITLE
Canvas Baseline

### DISCUSSION THREAD

**Discussion by timfi** [cetz-package/cetz#685] [Q&A]
Coming from Tikz, I found it quite difficult to align Cetz canvases such that specific content lies on the baseline of surrounding text. This makes building inline figures or other fancy typesetting very difficult. In Tikz, you can use node anchors in the [`baseline` option](https://tikz.dev/tikz-scopes#pgf./tikz/baseline) of the `tikzpicture`. Does Cetz already support something similar already and just haven't found it yet?

**Comment by johannes-wolf** [comment]
Fixed with #684.

**Reply by timfi** [reply]
This topic is something different to what the PR you mentioned adds. The "baseline" I'm referring to is not the baseline of individual node and text objects therein, but related to the canvas itself—see the image below for an illustration.

<img width="447" alt="Screenshot 2024-08-15 at 10 39 20" src="https://github.com/user-attachments/assets/f50e7521-e905-4d17-b2c2-70fdece3c6f4">

(Credits to [tikz.dev](https://tikz.dev/tikz-scopes#pgf./tikz/baseline) for the image.)

**Reply by timfi** [reply]
I'm currently playing around with this myself and am happy to port this discussion to an issue and/or open a related pull request.
